docs/src/main/sphinx/optimizer/cost-based-optimizations.rst

```diff
@@ -73,14 +76,50 @@ The valid values are:
 Capping replicated table size
 -----------------------------
 
-Join distribution type will be chosen automatically when join reordering strategy
-is set to ``AUTOMATIC`` or when join distribution type is set to ``AUTOMATIC``.
-In such case it is possible to cap the maximum size of replicated table via
-``join-max-broadcast-table-size`` config property (e.g. ``join-max-broadcast-table-size=100MB``)
-or via ``join_max_broadcast_table_size`` session property (e.g. ``set session join_max_broadcast_table_size='100MB';``)
-This allows to improve cluster concurrency and to prevent bad plans when CBO misestimates size of joined tables.
+The join distribution type is automatically chosen when the join reordering
+strategy is set to ``AUTOMATIC`` or when the join distribution type is set to
+``AUTOMATIC``. In both cases, it is possible to cap the maximum size of the
+replicated table with the ``join-max-broadcast-table-size`` configuration
```

Comment:
We should add this properties to  properties-optimizer.rst .. even if just a small section with link to here

Reply:
ping @colebow